docs/at_variables_reference.rst

```diff
@@ -0,0 +1,93 @@
+.. _AtVariables:
+
+@-Variables
+===========
+
+@-variables are placeholder variables that can be used in the ``experiments.yml`` file. They are enclosed
+by at signs, i.e. ``@...@`` and get resolved during runtime. Some use cases are:
+
+- allowing variations of experiments
+- accessing directories, which are created by simexpal during runtime
+- accessing values for keys specified in the ``experiments.yml``
```

Comment:
This is a bit unspecific. Can we maybe start with an example? One obvious use case is that you need @ variables to fill in the instance name.

Reply:
I removed those lines and added the ``@INSTANCE@``example